ZeroWasteFriendly/Support/SceneDelegate.swift

```diff
@@ -12,41 +12,41 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
 
+    lazy var coreDataStack = CoreDataStack(modelName: .modelName)
```

Comment:
another stack in SceneDelegate? 
I think you can remove one in App Delegate

Reply:
it could be also private or a shared singleton, so you don't have any var in scene

you can load 1 CD at one time so singleton can be ok

Reply:
you could have model inside that coredata stack, no need to pass it during initialization